resources/views/content/contributor.blade.php

```diff
@@ -13,10 +13,51 @@
             <imgix :src="$avatar_url" width="256" height="256" ratio="1:1" :alt="$login.' Avatar'" class="mx-auto rounded w-64 h-64" />
             <h2 class="text-6xl text-white font-bold">{{ $login }}</h2>
 
+            @if(isset($info['bio']))
+            <div class="py-4">  
+                <p class="mt-4">{{$info['bio']}}</p>
+            </div>
+            @endif
+
             <div class="flex flex-row flex-wrap justify-center">
                 <count-up icon="fa-box-heart" :value="count($packages)" label="packages" />
                 <count-up icon="fa-code-commit" :value="$commits" label="commits" />
             </div>
+
+            <div class="mx-auto max-w-3xl">
+                <div class="py-12 space-x-24 flex  justify-between">
+                    <div class="w-1/3">
+                        <i class="fas fa-location-circle fa-5x"></i> 
+                        <p class="mt-4">
+                            @if(isset($info['location']))
+                                {{$info['location']}}
+                            @else
+                                Whereabouts Unknown
+                            @endif
+                        </p>
+                    </div>
+                    <div class="w-1/3">
+                        <i class="fad fa-browser fa-5x"></i> 
```

Comment:
Please use the `fas` version and possibly the world/globe icon if the icon gets smaller the browser couldn't be easy to recognize.